### PR TITLE
Add order details links, reset fields on advanced configs close, balance validation, and add remove draft orders dialog

### PR DESCRIPTION
**Pull request by ribeirojose**

### CODE REVIEW

File: src/lib/tooltipDescriptions.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export enum TOOLTIP_DESCRIPTIONS {
   CURRENT_ORACLE_PRICE = "The price that will be monitored to trigger the order posting.",
   CURRENT_MARKET_PRICE = "The current price of the token pair in the market.",
   ORDER_CREATION = "The transaction that created this stop loss order.",
-  ORDER_HASH = "The onchain transaction for creating this order.",
+  ORDER_HASH = "The hash of the stop loss order on the Composable CoW framework. Each order for each user has a unique hash.",
```
Comment (ribeirojose) [inline src/lib/tooltipDescriptions.ts:7]: isn't this what they call order ID?

File: src/components/PriceInputCard.tsx [hunk 2]
```diff
@@ -79,11 +79,13 @@ function PriceInputCardComponent({
 
   const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const inputValue = parseFloat(e.target.value);
+    console.log(inputValue);
```
Comment (ribeirojose) [inline src/components/PriceInputCard.tsx:82]: 👀

File: src/components/DraftOrdersTab.tsx [hunk 3]
```diff
@@ -41,7 +42,20 @@ export function DraftOrdersTab() {
           <TableHeader className="bg-background">
             <TableRow>
               <TableHead className="rounded-tl-md">
-                <span className="sr-only">Select</span>
+                <Checkbox
+                  checked={
+                    selectedIds.length === draftOrders.length &&
+                    !!draftOrders.length
```
Comment (ribeirojose) [inline src/components/DraftOrdersTab.tsx:48]: what happens when the user manually deselects one of the orders?

File: src/components/AdvancedSettingsDialog.tsx [hunk 4]
```diff
@@ -78,8 +78,15 @@ export function AdvancedSettingsDialog() {
 
   const receiver = useWatch({ control, name: "receiver" });
 
+  const handleOpenChange = (isOpen: boolean) => {
+    if (!isOpen) {
+      reset(advancedSettings);
```
Comment (ribeirojose) [inline src/components/AdvancedSettingsDialog.tsx:83]: 🤔 why do we need this here?

File: src/lib/tooltipDescriptions.ts (hunk 1 above)
Comment (yvesfracari) [reply src/lib/tooltipDescriptions.ts:7]: No, the order ID is when it is posted on the Orderbook API. The hash is a parameter to find the order on the Composable CoW.

File: src/components/AdvancedSettingsDialog.tsx (hunk 4 above)
Comment (yvesfracari) [reply src/components/AdvancedSettingsDialog.tsx:83]: CoW Feedback:

> "When I update settings fields, then close the modal without pressing on the Save button, the values I've changes are remain to be displayed in the fields. 
However, the changes are not applied when place an order. This moght be confising to a user: settings show different info than order review modal. 
So here I can propose 2 options: 
> 1. Remove 'Save' button and auto-save all changes made on the Settings modal
> 2. OR reset changes to defaults when close settings modal without pressing on the Save button"

This is to implement the second solution.

File: src/components/DraftOrdersTab.tsx (hunk 3 above)
Comment (yvesfracari) [reply src/components/DraftOrdersTab.tsx:48]: It also deselects the select all checkbox

**Pull request merged**